benchmarks/dynamo/check_accuracy.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import os
+import sys
+import textwrap
+
+import pandas as pd
```

Comment:
This looks fairly duplicated with check_graph_breaks.py. Can we just fold these two scripts together?